packages/syft/tests/syft/service/sync/sync_resolve_single_test.py

```diff
@@ -86,3 +86,40 @@ def compute() -> int:
     client_low_ds.refresh()
     res = client_low_ds.code.compute(blocking=True)
     assert res == compute(blocking=True).get()
+
+
+def test_ignore_unignore_single(low_worker, high_worker):
+    low_client: DomainClient = low_worker.root_client
+    client_low_ds = get_ds_client(low_client)
+    high_client: DomainClient = high_worker.root_client
+
+    @sy.syft_function_single_use()
+    def compute() -> int:
+        return 42
+
+    compute.code = dedent(compute.code)
+
+    _ = client_low_ds.code.request_code_execution(compute)
+
+    diff = compare_clients(low_client, high_client)
+
+    assert len(diff.batches) == 2  # Request + UserCode
+    assert len(diff.ignored_batches) == 0
+
+    # Ignore usercode, request also gets ignored
+    res = diff[0].ignore()
```

Comment:
very ncie to have a test as well, if it could include the cases mentioned in the call that would be awesome